CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Changed
+- Removed types from `debug.traceback` arguments in the Lua 5.1 standard library

```

Comment:
Pedantic, but can you just quickly add some trailing whitespace here, so that there's a blank between this entry and the 0.6.0 header?